Sources/StreamChat/Database/DatabaseSession.swift

```diff
@@ -516,6 +562,64 @@ extension DatabaseSession {
                 log.warning("Failed to update message reaction in the database, error: \(error)")
             }
         }
+        
+        if let vote = payload.vote {
+            if payload.eventType == .pollVoteRemoved {
+                if let dto = try? pollVote(id: vote.id, pollId: vote.pollId) {
+                    delete(pollVote: dto)
+                }
+            } else if payload.eventType == .pollVoteChanged {
+                var voteUpdated = false
+                let userId = vote.userId ?? "anon"
+                if let optionId = vote.optionId, !optionId.isEmpty {
+                    let id = "\(optionId)-\(vote.pollId)-\(userId)"
```

Comment:
This appeared already once before, wondering if it makes sense to put this into a separate `static func` for making sure we never have issues with `id` creation.

Reply:
yes, good point, will add it